lib/typePolicies.ts

```diff
@@ -8,7 +8,7 @@ export const typePolicies: TypedTypePolicies = {
   Money: {
     fields: {
       localizedAmount: {
-        read(_, { readField }) {
+        read(_, { readField, args }) {
```

Comment:
TODO: format should take locale to format the price.
It might be a show stopper for extending local schema, because not all queries will need locale query argument (needs to be validated)